src/main/haskell/kore/test/Test/Kore/Unparser.hs

```diff
@@ -32,30 +32,6 @@ test_unparse =
                 )
             )
             "sort x{} []"
-        , unparseTest
```

Comment:
Why remove this test?

Reply:
The sentence that this test unparses is not valid Kore (650ce2a32f90f975cb3a5aab787de91187c33876).